### DISCUSSION TITLE
a more clear way to emphasize the links in pdf files

### DISCUSSION THREAD

**Discussion by toxdd** [sumatrapdfreader/sumatrapdf#3476] [Ideas]
As shown in the figure below, it would help readers note the links by coloring the links using light colors like lilac.

<img width="819" alt="Snipaste_2023-05-03_19-52-02" src="https://user-images.githubusercontent.com/49010736/235908878-5f19bd5d-a709-4963-988e-c1c345b3ca13.png">

**Comment by GitHubRulesOK** [comment]
coloured links can be written into a PDF by the PDF Writer.
SumatraPDF pre-release may not always show those, but should optionally show a surround in cyan (admittedly its pale but a good warning not to click in some cases.)
![image](https://user-images.githubusercontent.com/12754913/235984824-d8ee4bfc-d447-4ccd-83c3-9f74a497c533.png)
